app/views/events/_applicants_overview.html.erb

```diff
@@ -1,6 +1,10 @@
 <div class="container-fluid" id="applicants_overview">
   <h3><%= t '.title', title: @event.name %></h3>
-  <div id="free_places"><%= t '.free_places', count: @free_places %></div>
+  <% if @free_places == Float::INFINITY %>
```

Comment:
I'm not sure what you mean?
The unlimited flag is true, thus max_participants is infinity, thus free_places returns infinity

(this is also mostly a workaround until we know how to translate "infinity" automatically)